pytest.ini

```diff
@@ -1,2 +1,2 @@
 [pytest]
-addopts = -W error -ra -v --pyargs pytest_trio --verbose --cov
```

Comment:
What was the issue with -W error?

Reply:
MultiError is deprecated, and I didn't want to go for elaborate workarounds when I've got another PR ready to fix it (and re-enable -W error) so soon.